Gemfile

```diff
@@ -17,7 +17,7 @@ gem 'pundit'
 gem 'valid_email'
 gem 'rack-attack'
 gem 'ruby-saml'
-gem 'saml_idp', git: 'https://github.com/pkarman/saml_idp', branch: 'xmlenc'
+gem 'saml_idp', '~> 0.3.0'
```

Comment:
I wondered about that. The convention appears mixed. See `uglifier` and `coffee-rails` e.g.